### ISSUE TITLE
JetSmartFilters dropdown alphabetical order

### ISSUE THREAD

**Issue by chaser32419** [CrocoBlock/suggestions#3703]
I did check I even asked support it isn't a thing yet.

It seems like it would be a basic feature for something like a smart filter to have the ability to organize the dropdown in alphabetical order to make it easier for users to find what they are looking for when a website is large and has a lot of options in it. 

The feature, when a user creates a dropdown with the smart filter you give then the option to make the drop down alphabetical order A-Z.

**Before adding a new feature request, please, make sure you’ve checked:**

- if the feature is already added to the backlog or in work;
- if the feature is missing from the products.

**Please, describe the feature request, making it clear to understand.**

Specify the feature, the product to which it should be added, and why it is important to add it. 
Describe what effect or functionality you want to achieve as a result.

**Add an example of where this feature is already implemented. **

Provide a detailed explanation of where this feature should be used, and if there are any alternatives already. 

Feel free to add any videos, links, screenshots to illustrate the requested feature.

Please, note, that the feature requests can be accepted and processed only if they’re written in English.

**Comment by mayanksdudakiya** [comment]
Yes, Please this is very basic functionality to sort the filter values.
